js/faq.js

```diff
@@ -1,8 +1,37 @@
 const faqLinks = document.querySelectorAll('.faq_link');
 const currentLocation = window.location.hash;
 
+// to open accordian by pressing spacebar
+window.addEventListener('keydown', (e) => {
+  const ancTag = document.querySelector(
+    `a[href="${e?.target?.attributes?.href?.value}"]`,
+  );
+  if (e.key === ' ' && ancTag) {
```

Comment:
'    ' signify the case here?

Reply:
for open  accordion with space bar

Reply:
use e.code == "Space" ||      
      e.keyCode == 32 for spacebar key event